app/src/main/java/com/simply/birthdayapp/commonpresentation/components/image/NetworkImage.kt

```diff
@@ -0,0 +1,47 @@
+package com.simply.birthdayapp.commonpresentation.components.image
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import coil.request.ImageRequest
+import coil.transform.CircleCropTransformation
+import com.simply.birthdayapp.R
+
+@Composable
+fun NetworkImage(
+    modifier: Modifier = Modifier.size(72.dp),
+    url: String?,
```

Comment:
having nullable image url is  necessary?

Reply:
Yes , by default user don't have a images , and it is null 
![image](https://github.com/user-attachments/assets/63112480-5064-4fc1-a630-4bc4a6182c3c)